source/includes/kubernetes/_k8_deployments.md

```diff
@@ -0,0 +1,157 @@
+### Deployments
```

Comment:
The header nesting should be second level `##`, otherwise it will appear under the `k8_daemonsets` heading.

Reply:
This makes a lot of sense

Reply:
Done!